src/views/queries/queryResponses/viewResponseToScope3.tsx

```diff
@@ -0,0 +1,93 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { UUID } from '../../../models/strings.js'
+import { LinkButton, Page } from '../../common.js'
+type QueryStatus = 'resolved' | 'pending_your_input' | 'pending_their_input' | 'errored'
+
+interface Query {
+  id: UUID
+  company_name: string
+  query_type: string
+  updated_at: Date
+  status: QueryStatus
+  role: 'responder' | 'requester'
+  quantity: string
+  productId: string
+  emissions: string
+}
+@singleton()
+export default class Scope3CarbonConsumptionViewResponseTemplates {
+  //should be renamed to something more readable?
+  constructor() {}
+  public scope3CarbonConsumptionViewResponsePage = (query: Query) => {
+    return (
+      <Page
+        title="Veritable - Query Response"
+        activePage="queries"
+        heading="View response to your query"
+        headerLinks={[
+          { name: 'Queries', url: '/queries' },
+          { name: 'Query Request-someIdentifier', url: '/queries/query-response' },
+        ]}
+      >
+        <div class="connections header"></div>
+        <div class="card-body">
+          {' '}
+          <div class="container-scope3-carbon">
+            <div class="box1">
+              <h1>Scope 3 Carbon Consumption</h1>
+              <p class="query-text-carbon3-consumption">
+                A query for calculationg the total scope 3 carbon consumption for a given product or component.
+              </p>
+            </div>
+            <div class="box2">
+              <h2>Query Information</h2>
+              <div class="box3 ">
+                <table>
+                  <tr>
+                    <td>Supplier:</td>
+                    <td class="query-results-left-padding-table">{Html.escapeHtml(query.company_name)}</td>
+                  </tr>
+                  <tr>
+                    <td>ProductID:</td>
+                    <td class="query-results-left-padding-table">{Html.escapeHtml(query.productId)}</td>
+                  </tr>
+                  <tr>
+                    <td>Quantity:</td>
+                    <td class="query-results-left-padding-table">{Html.escapeHtml(query.quantity)}</td>
+                  </tr>
+                  <tr>
+                    <td>Query:</td>
+                    <td class="query-results-left-padding-table">
+                      Please provide details on the Scope 3 carbon emissions associated with the product.
+                    </td>
+                  </tr>
+                </table>
```

Comment:
since it's the table have not seen any data being rendered here, is it meant. to be?

Reply:
I chose to make it a table as it was the easiest in terms of layout (or it seemed the easiest to me) - if you look at the degn on the linked picture hopefully it'll make sense? 🤞

Reply:
In regards to that not sure if html <table> is the best option since we are using different approach elsewhere which is grid/flex, but this is something that we can review at the later stages of the `veritable-ui`.  Have not tried locally, but wonder how does it resize?

We also have some shared/generic containers e.g. for column/row so just thinking out loud if it would make sense to go through at some point in time

Reply:
Yeah I know what you mean I tried using the containers etc at first - but because it is different `rows` with a specific gap in between and each of the fields has a different length they became misaligned when resizing...I thought this way it actually resizes alright  
![Screenshot 2024-08-29 at 13 50 31](https://github.com/user-attachments/assets/865b25be-c0d8-467d-92c4-abe9e0f0b544)